components/layout/app-bar/app-bar.scss

```diff
@@ -1,24 +1,24 @@
 .icon {
-  height: 44px;
-  width: 44px;
-  color: white;
+  height: 42px;
+  width: 42px;
+  color: white !important;
```

Comment:
also can't the color be set on the icon? `!important` is always a last resort

Reply:
it could. done.